deps/chakrashim/src/jsrtdebug.cc

```diff
@@ -492,14 +492,16 @@ namespace jsrt {
     void *callbackState) {
     int frameIndex;
     JsValueRef scriptRef;
-    jsrt::StringUtf8 script;
     JsValueRef resultArray = JS_INVALID_REFERENCE;
     JsValueRef result = JS_INVALID_REFERENCE;
 
     if (argumentCount > 2 &&
       jsrt::ValueToInt(arguments[2], &frameIndex) == JsNoError &&
-      jsrt::ToString(arguments[1], &scriptRef, &script) == JsNoError) {
-      JsErrorCode errorCode = JsDiagEvaluateUtf8(script, frameIndex, &result);
+      JsConvertValueToString(arguments[1], &scriptRef) == JsNoError) {
```

Comment:
Here we don't need that, previously we were doing it to get the string (char*) as JsDiagEvaluateUtf8 used to take char*.